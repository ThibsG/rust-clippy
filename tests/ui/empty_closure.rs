#![warn(clippy::empty_closure)]

fn main() {
    // Lint
    std::thread::spawn(|| {});
    // No lint
    vec![0, 1, 2].iter().map(|_| {}).collect::<Vec<()>>();
}
