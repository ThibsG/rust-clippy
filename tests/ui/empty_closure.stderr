error: Empty closure may be removed
  --> $DIR/empty_closure.rs:5:24
   |
LL |     std::thread::spawn(|| {});
   |                        ^^^^^
   |
   = note: `-D clippy::empty-closure` implied by `-D warnings`

error: aborting due to previous error

